src/AddItemForm.svelte

```diff
@@ -0,0 +1,76 @@
+<script>
+  import { db } from "./firebase.js";
+
+  let newItemText;
+
+  function addItem() {
+    db.collection("grocery").add({
+      name: newItemText,
+    });
+    newItemText = "";
+  }
+</script>
+
+<form>
+  <div class="input-field">
+    <input
+      class="item-input"
+      title="Add-item"
+      type="text"
+      bind:value={newItemText}
+      placeholder="Item and quantity to buy"
+    />
+    <button
+      class="add-item-btn btn-white btn-animate"
+      on:click|preventDefault={addItem}>Add item</button
+    >
+  </div>
+</form>
+
+<style>
+  .item-input {
+    width: 250px;
+    flex-grow: 1;
+    background-color: var(--background-color-light);
+    border-radius: 5px;
+    color: #595959;
+    border: 1px solid #a6a6a6;
+    margin: 0 auto;
+    flex-grow: 1;
+    margin-top:10px;
+    box-shadow: none;
+  }
+
+  .item-input:focus {
+    outline: var(--action-btn-color) solid 2px;
+    box-shadow: 0 0 0 2px hsla(210, 100%, 50%, 0.5);
```

Comment:
@ClaraChaouat the only thing missing is the focus state in each interactive element (the buttons) :)

You can test it by using the keyboard `tab` and `shift + tab` to navigate between the page elements.

Reply:
@sandrina-p , so in the same logic would you recommend to also add focus to every list items?

Reply:
@sandrina-p is "testing" an app or a website as you used a sreenreader is something you recommend to do for every project? (I guess the answer is yes, but I'm asking :))